compiler/rustc_target/src/asm/powerpc.rs

```diff
@@ -47,6 +48,7 @@ impl PowerPCInlineAsmRegClass {
                 }
             }
             Self::freg => types! { _: F32, F64; },
+            Self::vreg => todo!(), //not quite sure :)
```

Comment:
I think this should be something like:
```rust
types! {
    vsx: I8, I16, I32, I64, F32, F64,
    altivec: VecI8(16), VecI16(8), VecI32(4), VecI64(2), VecF32(4), VecF64(2);
}
```
Altivec allows 128-bit vector types, and VSX allows using scalars in vector registers.

However this should be verified by writing asm tests that checks that LLVM is able to generate the correct code for these.